node/node_test.go

```diff
@@ -259,3 +259,9 @@ func TestDeadlineToggling(t *testing.T) {
 
 	test.TestDeadlineToggling(t, builder.MockSbBuilder, 2*time.Millisecond)
 }
+
+func TestVerifiedClientTopUp(t *testing.T) {
+	logging.SetLogLevel("storageminer", "FATAL")
+	logging.SetLogLevel("chain", "ERROR")
+	test.AddVerifiedClient(t, builder.Builder)
```

Comment:
This makes it use fake (much faster) proofs
```suggestion
	test.AddVerifiedClient(t, builder.MockSbBuilder)
```